### PR TITLE
Update README to account for zenhub migration

### PR DESCRIPTION
**Pull request by howardjohn**
Fixes https://github.com/istio/community/issues/384

### CODE REVIEW

**Review by ericvn — APPROVED** [review]

**Comment by howardjohn** [comment]
@linsun can you take look?

**Comment by howardjohn** [comment]
@linsun

**Review by linsun — COMMENTED** [review]
Is this migration already done?

File: README.md [hunk 1]
```diff
@@ -89,20 +89,14 @@ Envoy filters) that support authentication, authorization, and telemetry collect
...
+We use GitHub to track all of our bugs and feature requests. Each issue we track has a variety of metadata:
 
 - **Epic**. An epic represents a feature area for Istio as a whole. Epics are fairly broad in scope and are basically product-level things.
 Each issue is ultimately part of an epic.
 
 - **Milestone**. Each issue is assigned a milestone. This is 0.1, 0.2, ..., or 'Nebulous Future'. The milestone indicates when we
 think the issue should get addressed.
 
-- **Priority/Pipeline**. Each issue has a priority which is represented by the Pipeline field within GitHub. Priority can be one of
+- **Priority**. Each issue has a priority which is represented by the column in the [Prioritization](https://github.com/orgs/istio/projects/6) project. Priority can be one of
```
Comment (linsun) [inline README.md:100]: Is this the new board? :)

Comment (howardjohn) [reply README.md:100]: Yes, and the migration is done

**Review by linsun — APPROVED** [review]
New board looks nice!

**Pull request merged**